Vic2ToHoI4/Source/HOI4World/Map/ActualResources.cpp

```diff
@@ -0,0 +1,15 @@
+#include "ActualResources.h"
+#include "ParserHelpers.h"
+
+
+
+ActualResources::ActualResources(std::istream& theStream)
+{
+	registerKeyword(std::regex("[a-z]+"), [this](const std::string& resourceName, std::istream& theStream) {
+		const commonItems::singleDouble resourceAmount(theStream);
```

Comment:
double? do resources even come in double?

Reply:
In the end they're output as double, so through the whole chain that's what they are. And it gives us the possibility of tweaking our resource file to spread them out more if we wanted.